core/survey/_surveyValidator/nodeDefExpressionsValidator.js

```diff
@@ -116,7 +134,7 @@ const _validateNodeDefExpr = async (survey, nodeDef, dependencyType, expr) => {
 
 const _validateExpressionProp = (survey, nodeDef, dependencyType) => async (propName, item) => {
```

Comment:
is this async needed?

Reply:
yes because it call _validateNodeDefExpr that is async!